src/rect.cr

```diff
@@ -25,5 +25,13 @@ module SDL
 
     def initialize(@x, @y, @w, @h)
     end
+
+    def includes?(point : SDL::Point)
+      includes?(point.x, point.y)
+    end
+
+    def includes?(x : Int32, y : Int32)
+      x >= @x && x < @x + @w && y >= @y && y < @y + @h
```

Comment:
Crystal has a lovely sugar syntax for such comparisons:

 ```suggestion
      @x <= x < (@x + @w) && @y <= y < (@y + @h)
```

Reply:
Thanks, I updated the commit.